src/FixedDataTable.js

```diff
@@ -1177,7 +1177,9 @@ var FixedDataTable = createReactClass({
     }
 
     var columnResizingData;
-    if (props.isColumnResizing || (oldState && oldState.isColumnResizing)) {
```

Comment:
Just adding the props.isColumnResizing === undefined condition here so we only use oldState when isColumnResizing is not specified in the props.